src/components/Carousel/CategoryCard.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./CategoryCard.css";
+
+export default class CategoryCard extends React.Component {
+  render() {
+    const { data } = this.props;
+    return (
+      <>
+        {data.map((obj, index) => (
+          <div className="carousel-card" key={obj.id}>
+            <img src={obj.data.main_image.url} alt="" />
+            <div>
+              <h2>{obj.data.name}</h2>
+            </div>
+          </div>
+        ))}
```

Comment:
Aquí también podemos usar destructuración para evitar repetir tantas veces la expresión`obj`.

```suggestion
        {data.map(({ id, data: { main_image, name } }, index) => (
          <div className="carousel-card" key={id}>
            <img src={main_image.url} alt="" />
            <div>
              <h2>{name}</h2>
            </div>
          </div>
        ))}
```

Aquí hacemos destructuración dentro de la misma destructuración, se puede convertir en algo difícil de leer, así que puedes también hacerla dentro de la función también si no regresas el objeto tal cual.

```suggestion
        {data.map((obj, index) => {
          const { id } = obj;
          const { main_image, name } = obj.data;
          return <div className="carousel-card" key={id}>
            <img src={main_image.url} alt="" />
            <div>
              <h2>{name}</h2>
            </div>
          </div>
        })}
```